src/engine/NativeFunctionCall.ts

```diff
@@ -209,7 +209,7 @@ export class NativeFunctionCall extends InkObject {
           "NativeFunctionCall.CallBinaryListOperation op"
         );
       let result = op(v1.isTruthy ? 1 : 0, v2.isTruthy ? 1 : 0);
```

Comment:
I suggest we cast `result` with `Boolean` here, just to stay close to the source. I think BoolValue does a cast anyway, so not a huge deal.

Reply:
So `op` should already return a boolean value, since all the "ListBinaryOp" return a boolean. I kept it in line with what was there, but to get an equivalent behaviour, we would have to raise an exception if the return type isn't boolean rather than casting it the JavaScript way (it's a hard cast in C#). Happy to add it!

Reply:
Aaah right, I see. Nevermind, I think it's good as it is :)

Reply:
Haha, I added the new `asBooleanOrThrows` type assertion 😅